templates/controllers/grid/users/reviewer/readReview.tpl

```diff
@@ -10,46 +10,69 @@
  *}
 
 <script type="text/javascript">
-	$(function() {ldelim}
-		// Attach the form handler.
-		$('#readReviewForm').pkpHandler('$.pkp.controllers.form.AjaxFormHandler');
-	{rdelim});
+    $(function() {ldelim}
+        // Attach the form handler.
+        $('#readReviewForm').pkpHandler('$.pkp.controllers.grid.users.reviewer.ReadReviewHandler');
+    {rdelim});
 </script>
 
 <form class="pkp_form" id="readReviewForm" method="post" action="{url op="reviewRead"}">
 	<input type="hidden" name="reviewAssignmentId" value="{$reviewAssignment->getId()|escape}" />
 	<input type="hidden" name="submissionId" value="{$reviewAssignment->getSubmissionId()|escape}" />
 	<input type="hidden" name="stageId" value="{$reviewAssignment->getStageId()|escape}" />
-	<p>{translate key="editor.review.readConfirmation"}</p>
+    {if $reviewAssignment->getUnconsidered() == $smarty.const.REVIEW_ASSIGNMENT_UNCONSIDERED}
+        <p>{translate key="editor.review.readConfirmation"}</p>
+    {elseif $reviewAssignment->getDateCompleted() == ''}
```

Comment:
IMO it's a little cleaner to use...

    {elseif !$reviewAssignment->getDateCompleted()}

...rather than string comparisons. Also elsewhere in this file.